### DISCUSSION TITLE
Sound Card and Library PI4J V2

### DISCUSSION THREAD

**Discussion by manueltm24** [Pi4J/pi4j#159] [General]
I have an amplifier sound card that connects via GPIO. It uses specific GPIO PINs, my question is, does the card create any interruption with the library if I want to use a button?

**Comment by savageautomate** [comment]
It should be fine ... provided you don't try to reuse the same GPIO pins as the sound card :-) 
Pi4J should only attempt to configure the specific pins that you assign in your code ... it should not have an impact on other pins that you don't define in your Pi4J code.

Thanks, Robert

**Reply by manueltm24** [reply]
Well, I have the error: PIGPIO ERROR: PI_INIT_FAILED; pigpio initialisation failed. But when I play any music works well.

**Reply by manueltm24** [reply]
Exuse when not play a mp3
